grouper/fe/templates/base.html

```diff
@@ -21,33 +22,34 @@
 
     <body>
         <div id="head-nav" class="navbar navbar-default navbar-fixed-top">
-            <div class="navbar-header">
-                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
-                    <span class="fa fa-gear"></span>
-                </button>
-                <a class="navbar-brand" href="/" class="{{is_active('/')}}"><span>GROUPER.{{deployment_name}}</span></a>
-            </div>
-            <div class="navbar-collapse collapse">
-                <ul class="nav navbar-nav">
-                    <li class="{{is_active('/groups')}}"><a href="/groups">Groups</a></li>
-                    <li class="{{is_active('/users')}}"><a href="/users">Users</a></li>
-                    <li class="{{is_active('/permissions')}}"><a href="/permissions">Permissions</a></li>
-                    <li class="{{is_active('/help')}}"><a href="/help">Help</a></li>
-                </ul>
+            <div class="container">
```

Comment:
All of the other bootstrap examples I've seen wrap everything, including the navbar, inside a container. 

I've moved this so its outside its current parent div.